spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/integration/IntegrationAutoConfiguration.java

```diff
@@ -131,4 +143,47 @@ public IntegrationMBeanExporter integrationMbeanExporter() {
 
 	}
 
+	/**
+	 * Integration JDBC configuration.
+	 */
+	@Configuration
+	@ConditionalOnClass(JdbcMessageStore.class)
```

Comment:
This is redundant because of `@Conditional(IntegrationSchemaCondition.class)` below. Or that one should be pulled here.